src/main/java/com/mx/iwaconsolti/calculator/Resta.java

```diff
@@ -0,0 +1,13 @@
+package com.mx.iwaconsolti.calculator;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Resta {
+    private double Resultadoresta;
```

Comment:
This variable is not required since the result of the opertaion can be return directly from the method